src/Carbon/Carbon.php

```diff
@@ -1576,6 +1576,16 @@ public function isLeapYear()
     }
 
     /**
+     * Determines if the instance is in the current year
+     *
+     * @return bool
+     */
+    public function isCurrentYear()
+    {
+        return $this->year === static::now($this->tz)->year;
```

Comment:
You're right. I liked the readability and didn't realise it took an indirect route to `format`